portfolio/src/main/webapp/index.html

```diff
@@ -5,61 +5,82 @@
   <meta charset="UTF-8">
   <title>David Morley | Google Step Intern </title>
   <link rel="stylesheet" href="style.css">
-  <!--Add icon library for linkedIn, faceBook, etc-->
+  <!--Add icon library for linkedIn, FaceBook, etc.-->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   <script src="script.js"></script>
 </head>
 
 <body>
   <div id="content">
-    <div id="midTile">
-      <p class="Title">David Morley</p>
-      <img id ="Headshot" alt="David Morley Headshot" src="images/Headshot.jpg"></img>
-      <p>
-      <a target = "_blank" href="https://medium.com/@dmoe22"><i class="fa fa-dribbble"></i></a>
-      <a target = "_blank" href="https://linkedin.com/in/22dmoe"><i class="fa fa-linkedin"></i></a>
-      <a target = "_blank" href="https://m.facebook.com/david.morley.104855"><i class="fa fa-facebook"></i></a>
+    <div id="mid-tile">
+      <p class="title">David Morley</p>
+      <div class="portrait-tile">
+        <div class="portrait-tile-inner">
+          <div class="portrait-tile-front">
+            <img src="images/Headshot.jpg"></img>
+          </div>
+          <div class="portrait-tile-back">
+            <div>
+              <p> <b>College:</b> UCLA</p>
+              <p> <b>Degree:</b> Computer Science and Engineering</p>
+              <p> <b>Favorite Food:</b> Lasagna</p>
+              <p><b>Favorite T-shirt:</b> A teal t-shirt with a white penguin, it looks much cooler than it sounds.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <p id="icons">
+        <a target="_blank" href="https://medium.com/@dmoe22"><i class="fa
+        fa-dribbble"></i></a>
+        <a target="_blank" href="https://linkedin.com/in/22dmoe"><i class="fa
+        fa-linkedin"></i></a>
```

Comment:
I think these lines would be more readable unbroken. The style guide recommends wrapping only when it "significantly improves readability." When you do break lines, avoid breaking inside a class name. It makes it harder to do things like search for that class in the codebase.

https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

Reply:
Hmm yeah I'm going to have to play with the auto formatter then I'm afraid. Looks like the default is like 80 column width line wrap. Should I just try to turn this feature off or wrap at a much larger number, what would you recommend or perhaps there is some additional flag for this (though I didn't see one)?

Reply:
I'd recommend turning it off for HTML, and only using auto line wrapping for languages with a line length limit.